web/package.json

```diff
@@ -31,6 +31,8 @@
     "prop-types": "15.6.1",
     "react": "16.4.0",
     "react-dom": "16.4.0",
+    "react-firebaseui": "^3.0.4",
```

Comment:
Using `^` means that every time we install dependencies, the manager will grab the newest version - while semver ideally enforces that no breaking changes will occur, it's probably best to lock it down by removing `^` and only upgrading explicitly. This helps avoid `lock`file diffs polluting commit history.

Reply:
do we want to do that for every package in this file?